src/current/cockroachcloud/managing-cmek.md

```diff
@@ -27,6 +27,79 @@ This section shows how to enable CMEK on a CockroachDB {{ site.data.products.adv
   <button class="filter-button" data-scope="gcp">GCP</button>
 </div>
 
+### Before you begin
+
+<section class="filter-content" markdown="1" data-scope="aws">
+
+1. Make a note of your {{ site.data.products.cloud }} organization ID in the [Organization settings page](https://cockroachlabs.cloud/settings).
+1. Find your {{ site.data.products.advanced }} cluster ID. From the CockroachDB {{ site.data.products.cloud }} console [Clusters  list](https://cockroachlabs.cloud/clusters), click the name of a cluster to open its **Cluster Overview** page. From the page's URL copy the cluster ID, which is the portion of the URL before `/overview/`.
+1. Use the cluster ID to find the cluster's associated cross-account IAM role, which is managed by CockroachDB {{ site.data.products.cloud }}.
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{YOUR_CLUSTER_ID} \
+      --header 'Authorization: Bearer {YOUR_API_KEY}' | jq .account_id
+    ~~~
+
+    In the response, verify that the `id` field matches the cluster ID you specified, then make a note of the `account_id`, which is the ID for the cross-account IAM role.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="gcp">
+
+1. Make a note of your {{ site.data.products.cloud }} organization ID in the [Organization settings page](https://cockroachlabs.cloud/settings).
+1. Find your {{ site.data.products.advanced }} cluster ID. From the CockroachDB {{ site.data.products.cloud }} console [Clusters  list](https://cockroachlabs.cloud/clusters), click the name of a cluster to open its **Cluster Overview** page. From the page's URL copy the cluster ID, which is the portion of the URL before `/overview/`.
+1. Use the cluster ID to find the cluster's associated GCP Project ID, which is managed by CockroachDB {{ site.data.products.cloud }}. Query the `clusters/` endpoint of the CockroachDB {{ site.data.products.cloud }} API:
+
+    {% include_cached copy-clipboard.html %}
+    ```shell
+    CLUSTER_ID= #{ your cluster ID }
+    API_KEY= #{ your API key }
+    curl --request GET \
+      --url https://cockroachlabs.cloud/api/v1/clusters/${CLUSTER_ID} \
+      --header "Authorization: Bearer ${API_KEY}"
+    ```
+
+    In the response, verify that the `id` field matches the cluster ID you specified, then make a note of the following:
+      - `account_id`: the GCP project ID.
+      - `regions`/`name`: one entry for each of the cluster's regions. CMEK must be configured in each of a cluster's regions.
+
+      ```json
+      {
+        "id": "blahblahblah-9ebd-43d9-8f42-589c9e6fc081",
+        "name": "docs-rule",
+        "cockroach_version": "v22.1.1",
+        "plan": "DEDICATED",
+        "cloud_provider": "GCP",
+        "account_id": "docs-rule-123",
```

Comment:
I think it would be worth it to have an actual example, like `crl-prod-xyz`

Reply:
Done